deploy-manage/remote-clusters/eck-remote-clusters-to-external.md

```diff
@@ -0,0 +1,355 @@
+
+For other remote cluster scenarios with ECK, refer to [Remote clusters on ECK](./eck-remote-clusters-landing.md#eck-rcs-setup).
+
+% refine this note
+:::{note}
+This guide uses API key authentication as the [security model](./security-models.md), which is the recommended option and replaces the deprecated TLS certificate–based model.  
+
+If you need to configure TLS certificate authentication for this scenario, refer to the steps in [Connect from an external cluster](./eck-remote-clusters-from-external.md) and create the remote in the opposite direction. The mutual-TLS trust setup steps the similar.
+:::
+
+## Allow the remote connection [ec_allow_the_remote_connection_4]
+
+% consider replacing this intro
+:::{include} _snippets/apikeys-intro.md
+:::
+
+### Enable the remote cluster server interface on the remote cluster [enable-rcs]
+
+Follow the steps corresponding to the deployment type of your remote cluster:
+
+:::::::{applies-switch}
+
+::::::{applies-item} ess:
+If the remote cluster is part of an {{ech}} deployment, the remote cluster server is enabled by default and it uses a publicly trusted certificate provided by the platform proxies. Therefore, you can skip this step.
+::::::
+
+::::::{applies-item} ece:
+If the remote cluster is part of an {{ece}} deployment, the remote cluster server is enabled by default, and secured with TLS certificates.
+
+Depending on the type of certificate used by the ECE proxies or load-balancing layer, the local cluster requires the associated certificate authority (CA) to establish trust:
+
+* If your ECE proxies use publicly trusted certificates, no additional CA is required.
+
+* If your ECE proxies use certificates signed by a private CA, retrieve the root CA from the [ECE Cloud UI](/deploy-manage/deploy/cloud-enterprise/log-into-cloud-ui.md):
+
+  1. In the remote ECE environment, go to **Platform > Settings > TLS certificates**.
+
+  2. Under **Proxy**, select **Show certificate chain**.
+
+  3. Click **Copy root certificate** and paste it into a new file. The root certificate is the last certificate shown in the chain.
+
+  4. Save the file as `.crt`, and keep it available for the trust configuration on the local cluster.
+::::::
+
+::::::{applies-item} eck:
+If the remote cluster is managed by a different ECK operator, it must be prepared to accept incoming connections.
+
+1. **Enable the remote cluster server**
+
+    :::{include} _snippets/eck_rcs_enable.md
+    :::
+
+2. **Expose the remote cluster server interface**
+
+    :::{include} _snippets/eck_rcs_expose.md
+    :::
+
+3. **Retrieve the certificate authority (CA)**
+
+    :::{include} _snippets/eck_rcs_retrieve_ca.md
+    :::
+
+::::::
+::::::{applies-item} self:
+
+1. **Enable and secure the remote cluster server**
```

Comment:
as discussed, I think it would be nice if these used headings instead of having the procedure go really deep